### PR TITLE
feat(workloads): add a workloads subcommand

### PR DESCRIPTION
**Pull request by ctrombley**
This PR adds workloads to the CLI.

Output is prettified and sent to stdout for get, list, create, and duplicate methods only.  Otherwise success/failure is indicated on stderr and command exit status.

### CODE REVIEW

File: internal/workload/command_workload.go [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+	err = cmdList.MarkFlagRequired("accountId")
+	if err != nil {
+		log.Error(err)
+	}
+
+	// Create
+	Command.AddCommand(cmdCreate)
+	cmdCreate.Flags().IntVarP(&accountID, "accountId", "a", 0, "the New Relic account ID where you want to create the workload")
+	cmdCreate.Flags().StringVarP(&name, "name", "n", "", "the name of the workload")
+	cmdCreate.Flags().StringSliceVarP(&entityGUIDs, "entityGuid", "e", []string{}, "the list of entity Guids composing the workload")
```
Comment (jthurman42) [inline internal/workload/command_workload.go]: `entityGuids` as the input name? (takes a list)

File: internal/workload/command_workload.go [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+	err = cmdCreate.MarkFlagRequired("name")
+	if err != nil {
+		log.Error(err)
+	}
+
+	// Update
+	Command.AddCommand(cmdUpdate)
+	cmdUpdate.Flags().StringVarP(&guid, "guid", "g", "", "the GUID of the workload you want to update")
+	cmdUpdate.Flags().StringVarP(&name, "name", "n", "", "the name of the workload")
+	cmdUpdate.Flags().StringSliceVarP(&entityGUIDs, "entityGuid", "e", []string{}, "the list of entity Guids composing the workload")
```
Comment (jthurman42) [inline internal/workload/command_workload.go]: `entityGuids` again

File: internal/workload/command_workload.go [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+	cmdUpdate.Flags().StringSliceVarP(&entitySearchQueries, "entitySearchQuery", "q", []string{}, "a list of search queries, combined using an OR operator")
```
Comment (jthurman42) [inline internal/workload/command_workload.go]: Singular flag accepting plural

File: internal/workload/command_workload.go [hunk 4]
```diff
@@ -0,0 +1,303 @@
...
+	if err != nil {
+		log.Error(err)
+	}
+
+	// Create
+	Command.AddCommand(cmdCreate)
+	cmdCreate.Flags().IntVarP(&accountID, "accountId", "a", 0, "the New Relic account ID where you want to create the workload")
+	cmdCreate.Flags().StringVarP(&name, "name", "n", "", "the name of the workload")
+	cmdCreate.Flags().StringSliceVarP(&entityGUIDs, "entityGuid", "e", []string{}, "the list of entity Guids composing the workload")
+	cmdCreate.Flags().StringSliceVarP(&entitySearchQueries, "entitySearchQuery", "q", []string{}, "a list of search queries, combined using an OR operator")
```
Comment (jthurman42) [inline internal/workload/command_workload.go]: Singular flag accepting plural

File: internal/workload/command_workload.go (hunk 1 above)
Comment (ctrombley) [reply internal/workload/command_workload.go]: Usage is `--entityGuid abc --entityGuid xyz`, similar to other commands, I just followed the convention here.

Comment (ctrombley) [reply internal/workload/command_workload.go]: Delete tags command for reference: https://github.com/newrelic/newrelic-cli/blob/master/internal/entities/command_tags.go#L234

Comment (jthurman42) [reply internal/workload/command_workload.go]: Ah, gotcha.

Comment (ctrombley) [reply internal/workload/command_workload.go]: Resolved offline.

File: internal/workload/command_workload.go (hunk 4 above)
Comment (ctrombley) [reply internal/workload/command_workload.go]: Resolved offline.

File: internal/workload/command_workload.go (hunk 2 above)
Comment (ctrombley) [reply internal/workload/command_workload.go]: Resolved offline.

File: internal/workload/command_workload.go (hunk 3 above)
Comment (ctrombley) [reply internal/workload/command_workload.go]: Resolved offline.

**Pull request merged**